via/static/scripts/video_player/config.ts

```diff
@@ -0,0 +1,32 @@
+import type { Transcript } from './components/Transcript';
```

Comment:
This module follows the same pattern as `frontend_apps/config.ts` in the LMS repo, but is much simpler since we only have one app with very simple configuration.